lib/onebox/engine/amazon_onebox.rb

```diff
@@ -11,11 +11,22 @@ class AmazonOnebox
       include HTML
 
       always_https
-      matches_regexp(/^https?:\/\/(?:www\.)?(?:smile\.)?(amazon|amzn)\.(?<tld>com|ca|de|it|es|fr|co\.jp|co\.uk|cn|in|com\.br|com\.mx)\//)
+      matches_regexp(/^https?:\/\/(?:www\.)?(?:smile\.)?(amazon|amzn)\.(?<tld>com|ca|de|it|es|fr|co\.jp|co\.uk|cn|in|com\.br|com\.mx|nl|pl|sa|sg|se|com\.tr|ae)\//)
```

Comment:
Is it possible to get the domain from the `amazon_domains` method in core? That way we have one source of truth?

Reply:
I have made a note to do this as a future change... It's possible that the `amazon_domains` in core may not be helping us solve the problem and could be removed.